pom.xml

```diff
@@ -1067,7 +1093,7 @@
               </execution>
 
               <execution>
-                <phase>generate-sources</phase>
+                <phase>compile</phase>
```

Comment:
:bug: this breaks anything using

```xml
<resources>
    <resource>
        <directory>src/main/resources</directory>
    </resource>
    <resource>
        <directory>target/generated-adjuncts</directory>
    </resource>
</resources>
```

as the `target/generated-adjuncts` is not created until after `maven-resources-plugin:2.7:resources` has run:

```
skip non existing resourceDirectory …/target/generated-adjuncts
```

Thus the plugin JAR omits the adjuncts, and at runtime you get

```
… org.kohsuke.stapler.framework.adjunct.AdjunctsInPage findNeeded
WARNING: No such adjunct found: …
org.kohsuke.stapler.framework.adjunct.NoSuchAdjunctException: Neither ….css, .js, .html, nor .jelly were found
	at org.kohsuke.stapler.framework.adjunct.Adjunct.<init>(Adjunct.java:125)
	at org.kohsuke.stapler.framework.adjunct.AdjunctManager.get(AdjunctManager.java:147)
	at org.kohsuke.stapler.framework.adjunct.AdjunctsInPage.findNeeded(AdjunctsInPage.java:161)
	at org.kohsuke.stapler.framework.adjunct.AdjunctsInPage.generate(AdjunctsInPage.java:113)
	at org.kohsuke.stapler.jelly.AdjunctTag.doTag(AdjunctTag.java:83)
	at …
```

Part of this can be seen for example by updating `pipeline-stage-view-plugin/pom.xml` to 2.6+ (CC @svanoort), though in that case the final JAR _does_ contain the adjuncts, for reasons I am still studying.

CC @andresrc.